package.json

```diff
@@ -22,19 +22,27 @@
   "scripts": {
     "commit": "cz",
     "import": "node ./scripts/import.js",
-    "build": "node ./scripts/build.js",
+    "build": "node ./scripts/build.js && node ./scripts/i18n.js",
+    "i18n": "node ./scripts/i18n.js",
+    "i18n:clear": "node scripts/i18n-helpers/clear-locales.js",
+    "i18n:get-sorted-locales": "node scripts/i18n-helpers/sort-locales.js",
```

Comment:
When is it time to run those scripts? I mean both `i18n:clear` and this one

Reply:
I've added some comments in the script files:

```
/**
 * Helper to remove locales folder
 * Test that localization files in raw folder are made correctly with the i18n script
 */
```
This one is nice to keep when we are making changes to i18n, and want to test the the locales folder will be made correctly. 

```
/**
 * Helper script to clean up and get a sorted default icon descriptions
 */
```

This one is useful when we are adding new translations, and want to sort the long file that holds all default English translations. I also used chatGPT to generate this file, and chatgpt (or I 😛)  did some mistakes (so there were some duplicates that I wanted to remove.

Reply:
But, maybe I should remove it from the scripts in package.json. Because they wont be used that much. Wdyt?